GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,90 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	@MODULE[ModuleSimpleAdjustableFairing]
+/// 	{
+/// 		ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// 	}
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
```

Comment:
Doesn't RO normally put these things in the part itself?

Reply:
Moved tag to part level